cmd/keytransparency-signer/main.go

```diff
@@ -86,17 +79,12 @@ func main() {
 	tmap := trillian.NewTrillianMapClient(mconn)
 
 	// Connection to append only log
-	tlog, err := config.LogClient(*logID, *logURL, *logPubKey)
+	lconn, err := grpc.Dial(*logURL, grpc.WithInsecure())
```

Comment:
Is this on the same machine? If yes, will it always be on the same machine? If not, why did we switch to an insecure connection?